src/librustc/hir/lowering.rs

```diff
@@ -571,6 +574,21 @@ impl<'a> LoweringContext<'a> {
         })
     }
 
+    fn expect_full_def_from_use(&mut self, id: NodeId) -> Vec<Def> {
```

Comment:
Oh, it turns out i can! I was scarred because i tried doing that to a version of `lower_path` (which i wound up taking out when we moved to the AST modification) and wound up with exotic lifetime errors due to the introduction of a `&ast::Path` with an unaccounted lifetime. But this one doesn't have to use anything external for its mapping, so it works out.